### ISSUE TITLE
(Android) Instabug database init exception

### ISSUE THREAD

**Issue by peacechen** [Instabug/Instabug-React-Native#223]
### Steps to Reproduce the Problem
Runtime error in the wild.  No repro known.

### Expected Behavior
No exception.

### Actual Behavior

Sentry reported this crash out in the field:

> IllegalStateException
> DatabaseManager is not initialized, call init(..) method first.
> com.instabug.library.internal.storage.cache.db.DatabaseManager in getInstance at line 24
> com.instabug.library.logging.b in a at line 20
> com.instabug.library.model.NetworkLog in insert at line 166
> com.instabug.reactlibrary.RNInstabugReactnativeModule in networkLog at line 2008
> java.lang.reflect.Method in invoke
> com.facebook.react.bridge.JavaMethodWrapper in invoke at line 372
> com.facebook.react.bridge.JavaModuleWrapper in invoke at line 160
> com.facebook.react.bridge.queue.NativeRunnable in run
> android.os.Handler in handleCallback at line 751
> android.os.Handler in dispatchMessage at line 95
> com.facebook.react.bridge.queue.MessageQueueThreadHandler in dispatchMessage at line 29
> android.os.Looper in loop at line 154
> com.facebook.react.bridge.queue.MessageQueueThreadImpl$3 in run at line 192
> java.lang.Thread in run at line 762
> 

### Instabug integration code
```
Instabug.startWithToken(Config.INSTABUG_KEY, [Instabug.invocationEvent.shake]);
Instabug.setWelcomeMessageMode(Instabug.welcomeMessageMode.disabled);
Instabug.disable(); // Android only
Instabug.setShakingThresholdForIPhone(10000, 10000); // iOS; effectively disable
BugReporting.setInvocationOptions([BugReporting.invocationOptions.commentFieldRequired]);
```

### SDK Version
Instabug-React-Native 8.0.17

### React Native, iOS and Android Versions
RN 0.56.1
Android 7.0

### Device Model
Samsung Note 5 (SM-N920V)

**Comment by peacechen** [comment]
A second bug report arrived, this time on an Essential PH-1 running Android 9.  I personally observed this when opening the app.  Our app started, showed the splash screen, then died with no visible error.  The Sentry bug report is the only indication of this error.

**Comment by salmamali** [comment]
hello @peacechen,
This error was fixed in version 8.0.18.

**Comment by peacechen** [comment]
Thanks @salmamali 

It would really help if the release notes were more informative.  Critical bug fixes like this should be listed so that we can make a decision whether an update is needed.  In this case since it's a native code change, we have to plan for an App/Play Store release instead of the quicker CodePush update.
https://github.com/Instabug/Instabug-React-Native/releases
